### PR TITLE
1193: (imp) Update logout button look and feel

### LINKED ISSUE #1193: Change UI for login/logout button
The login and logout button  look similar. There is no cue that you are in a particular state. It'd be great to have a visual indication by maybe your initials and a logout button or something

### PR DESCRIPTION
**Pull request by patricksimonian**
## Summary
Fixes #1193.

Just changed the logout button to look like a link so that is more differentiable from the login button

### CODE REVIEW

File: app-web/src/components/Navbar/Navbar.js [hunk 1]
```diff
@@ -91,7 +91,7 @@ export const NavList = styled.ul`
 
 const AuthContainer = styled.div`
   padding: 10px;
-  background-color: #fafafa;
+  border-bottom: 1px solid #bbc7d5;
```
Comment (patricksimonian) [inline app-web/src/components/Navbar/Navbar.js]: why remove the bg?

Comment (jas29) [reply app-web/src/components/Navbar/Navbar.js]: The logout button looks drastically different than the login button, after removing the background. 👍

**Comment by patricksimonian** [comment]
@jas29 i'll promote this to test

**Comment by patricksimonian** [comment]
@jas29 just cleaning up this PR and merging soon. Please be aware you'll need to rebase any prs you are working on with master

**Pull request merged**